app/services/mentors/create.rb

```diff
@@ -31,25 +31,18 @@ def call
         Mentors::AddToSchool.call(school: school_cohort.school, mentor_profile:)
       end
 
-      unless sit_validation
-        ParticipantMailer.with(participant_profile: mentor_profile).participant_added.deliver_later
-        mentor_profile.update_column(:request_for_details_sent_at, Time.zone.now)
-        ParticipantDetailsReminderJob.schedule(mentor_profile)
```

Comment:
There's a ton of these jobs scheduled in prod we should probably remove at the same time